Dfe.Academies.Academisation.Data/ProjectAggregate/CreateInvoluntaryProjectCommand.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Dfe.Academies.Academisation.Core;
+using Dfe.Academies.Academisation.Domain.ProjectAggregate;
+using Dfe.Academies.Academisation.IData.ProjectAggregate;
+using Dfe.Academies.Academisation.IDomain.ProjectAggregate;
+
+namespace Dfe.Academies.Academisation.Data.ProjectAggregate
+{
+	public class CreateInvoluntaryProjectCommand : ICreateInvoluntaryProjectCommand
```

Comment:
I think the original intention is this class would live in the service project. And the service project calls into the data and domain projects.
Though to be honest much of this has gone out the window since then.
![image](https://user-images.githubusercontent.com/109524605/217864877-514f8315-04fe-4596-afb4-c1ea7f8b9cee.png)

Reply:
Yeah I tried to search for a consistent approach within the codebase and struggled so I followed the pattern Tim used for the ProjectAddNotes slices.